src/Uno.UWP/Devices/Enumeration/Internal/Providers/ProximitySensor/ProximitySensorDeviceClassProvider.cs

```diff
@@ -0,0 +1,61 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using Android.Hardware;
+using Uno.Devices.Enumeration.Internal;
+using Uno.Devices.Sensors.Helpers;
+
+namespace Windows.Devices.Enumeration.Internal.Providers.ProximitySensor;
+
+internal class ProximitySensorDeviceClassProvider : IDeviceClassProvider
+{
+	public bool CanWatch => false;
+
+#pragma warning disable CS0067
+	public event EventHandler<DeviceInformation>? WatchAdded;
+
+	public event EventHandler<DeviceInformation>? WatchEnumerationCompleted;
+
+	public event EventHandler<DeviceInformationUpdate>? WatchRemoved;
+
+	public event EventHandler<object>? WatchStopped;
+
+	public event EventHandler<DeviceInformationUpdate>? WatchUpdated;
```

Comment:
Should they be invoked properly?

Reply:
The watcher is marked as not supported, so they are not used. I agree that the API should be improved later to separate interface though.